androidapp/app/src/main/assets/weather.json

```diff
@@ -1,3 +1,94 @@
 {
-  "TODO" : "FILL THIS WITH TEST DATA ONCE WE KNOW WHAT BACKEND IS PASSING AND WHAT IT LOOKS LIKE"
+  "sensor": {
+    "temperature": 0.00, // degrees C, check w/brian
+    "humidity": 0.00, // percent, check w/brian
+  },
+  "nws": {
+    "current": {
+      "temperature": 6.1111111111111427, // degrees C
+      "maxTemperature": 15.5555555555556, // degrees C
+      "minTemperature": 4.4444444444444571, // degrees C
+      "relativeHumidity": 93, // percent
+      "windChill": 2.7777777777777715, // degrees C
+      "windDirection": 119.99999999999999, // angle in degrees
+      "windSpeed": 4.6299960000000002, // meters per second
+      "windGust": 8.2311040000000002, // meters per second
+      "weather": { // items in here may be null
+        "coverage": "definite", // string
+        "weather": "rain_showers", // string
+        "intensity": "light", // string
+        "visibility": 8.0467200000000005 // kilometers
+      },
+      "probabilityOfPrecipitation": 92 // percent
+    },
+    // these forecasts were from before i switched them to si units, so assume si units
+    "daily": {
+      "periods": [ // i'm just copying these directly not removing stuff
+        {
+          "number": 1,
+          "name": "This Afternoon",
+          "startTime": "2020-03-13T17:00:00-04:00",
+          "endTime": "2020-03-13T18:00:00-04:00",
+          "isDaytime": true,
+          "temperature": 60,
+          "temperatureUnit": "F",
```

Comment:
actual units will be °C, this api response was fetched before i specified si in the calls in the daemon.